flutter/lib/sentry_flutter.dart

```diff
@@ -9,4 +13,97 @@ mixin SentryFlutter {
     final String version = await _channel.invokeMethod('getPlatformVersion');
     return version;
   }
+
+  static void _flutterErrorIntegration(Hub hub, SentryOptions options) {
+    final defaultOnError = FlutterError.onError;
+
+    FlutterError.onError = (FlutterErrorDetails errorDetails) async {
+      options.logger(
+          SentryLevel.debug, 'Capture from onError ${errorDetails.exception}');
+
+      // TODO: create mechanism
+
+      await hub.captureException(
+        errorDetails.exception,
+        stackTrace: errorDetails.stack,
+      );
+
+      // call original handler
+      if (defaultOnError != null) {
+        defaultOnError(errorDetails);
+      }
+
+      // do we need this?
+      // print('Capture from FlutterError ${details.exception}');
+      // Zone.current.handleUncaughtError(details.exception, details.stack);
+    };
+  }
+
+  static void _isolateErrorIntegration(Hub hub, SentryOptions options) {
+    final receivePort = RawReceivePort(
+      (dynamic error) async {
+        options.logger(SentryLevel.debug, 'Capture from IsolateError $error');
+
+        // is it a list? cond. error.length != 2 is probably wrong
```

Comment:
It's documented. It's always 2 items.

`The errors are sent back as two elements lists. `

https://api.flutter.dev/flutter/dart-isolate/Isolate/addErrorListener.html

Should it be `== 2` ? We expect exactly 2 items here

Reply:
yep, i fixed this already and linked the docs as well